src/engine/meta/meta_blks_mgr.cpp

```diff
@@ -399,6 +403,13 @@ void MetaBlkMgr::register_handler(const meta_sub_type type, const meta_blk_found
     HS_LOG(INFO, metablk, "[type={}] registered with do_crc: {}", type, do_crc);
 }
 
+void MetaBlkMgr::create_sobject(meta_sub_type type) {
+    auto sm{HomeStoreBase::safe_instance()->sobject_mgr()};
+    auto meta_blk_obj =
```

Comment:
don't need to save to meta_blk_obj, right? 
Also we don't need a function here, i think, you can directly call HomeStoreBase::safe_instance()->sobject_mgr()->create_object(...); from line:266.

Reply:
right